src/db/utils.js

```diff
@@ -0,0 +1,10 @@
+const customRequireValidator = function (next) {
```

Comment:
Not sure about this name.

`lanEntryExistsValidator` (thats also awful but maybe better?)